packages/teleport-publisher-vercel/src/crypto.js

```diff
@@ -0,0 +1,4 @@
+export const encryptFileData = async (fileData) =>  {
+    const digest = await crypto.subtle.digest('SHA-1', fileData)
```

Comment:
We can do direct return here like:
`return crypto.subtle.digest('SHA-1', fileData)`

And if we target browsers, shouldn't we use `window.crypto.` ?